packages/discussions/src/channels.ts

```diff
@@ -7,7 +7,14 @@ import {
   IRemoveChannelOptions,
   IChannel,
   IPagedResponse,
-  IRemoveChannelResponse
+  IRemoveChannelResponse,
+  IFetchOptOutOptions,
```

Comment:
We should probably make these interfaces names a bit more consistent. I'd recommend the following:

```
  IFetchChannelNotificationOptOutOptions,
  ICreateChannelNotificationOptOutOptions,
  IRemoveChannelNotificationOptOutOptions,
  IRemoveChannelNotificationOptOutResult,
  IRemoveChannelActivityOptions,
  IRemoveChannelActivityResult,
  IChannelNotificationOptOut,
```